src/libs/antares/study/parameters.cpp

```diff
@@ -425,7 +426,7 @@ static bool SGDIntLoadFamily_H(Parameters& d,
     if (key == "hydro-pricing-mode")
     {
         auto hpricing = StringToHydroPricingMode(value);
-        if (hpricing != ucUnknown)
+        if ((int)hpricing != (int)ucUnknown)
```

Comment:
We should discuss this test. 
We are comparing 2 differents enum : 

```
enum HydroPricingMode
{
    hpHeuristic = 0,
    hpMILP, // mixed integer linear problem
    hpUnknown,
};
```

and 

```
enum UnitCommitmentMode
{
    ucHeuristic = 0,
    ucMILP, // mixed integer linear problem
    ucUnknown,
};
```

I guess `ucUnknown `should be replaced by `hpUnknown`.  We could then replace the `auto `by `HydroPricingMode`